www/caddy/pkg-descr

```diff
@@ -27,15 +30,16 @@ Plugin Changelog
 
 1.6.2
 
-* Add Authentik as authentication provider (contributed by Tim-Sc)
+* Add: Authentik as authentication provider (contributed by Tim-Sc)
+* Add: Feature Preview - Layer4 routing to proxy traffic without TLS termination. Can be enabled in advanced mode of "General Settings".
 
 1.6.1
 
-Add: Run Caddy as "www" user and group, by enabling "Disable Superuser" in General Settings.
-Cleanup: Validations in general.volt are all appended to their form keys, instead of triggering a Bootstrap Dialog.
-Change: Description Fields are optional now, reducing the steps needed to create new entries.
-Add: Shortcuts to add new Domains and Handlers.
-Change: Subdomain Tab only appears when a wildcard domain has been configured.
+* Add: Run Caddy as "www" user and group, by enabling "Disable Superuser" in General Settings.
+* Cleanup: Validations in general.volt are all appended to their form keys, instead of triggering a Bootstrap Dialog.
+* Change: Description Fields are optional now, reducing the steps needed to create new entries.
+* Add: Shortcuts to add new Domains and Handlers.
+* Change: Subdomain Tab only appears when a wildcard domain has been configured.
```

Comment:
just to nitpick for digestion purposes ordering for add/cleanup/change blocks would be beneficial

Reply:
Cleanup up the whole file. :D